src/modules/editingTask.js

```diff
@@ -15,6 +15,9 @@ function finishEditing(event) {
 }
 
 function editingTask(event) {
+  document.querySelectorAll('.editing-background').forEach((element) => {
+    element.classList.remove('editing-background');
+  });
   // console.log(event.currentTarget.classList[0]);
```

Comment:
- [OPTIONAL] Please it will be nice to remove logging statements from your code base as it makes your app vulnerable to attacks.